types/unist/index.d.ts

```diff
@@ -1,40 +1,95 @@
 // Type definitions for Unist 2.0
-// Project: https://github.com/syntax-tree/unist
+// Project: https://github.com/syntax-tree/unist-ts
```

Comment:
I'm confused - the type definitions for unist are distributed both separately and on DefinitelyTyped?

Reply:
Well, I'm not sure why the unist-ts project was created initially. I think the idea was to keep the unist definition language agnostic, and probably the project served as a suggestion to show how the types may look like. 

Now that there is both a @types/unist and a unist-ts, I was going to suggest to @wooorm to delete the types from unist-ts and change the project purpose to be a documentation so people could see how to use the unified ecosystem using TypeScript.

I know that he wants to avoid adding the types to each project as it would cause more burden to the maintainers, that's why we are adding the definitions here.

Reply:
That's a little strange, but it can be sorted out offline.  For now, I'd suggest reverting the URL if that's the underlying project that these types describe.

Reply:
@amcasey Hey, author of unist here. The weird part is that the underlying project (unist) is a spec. There’s no code. I’m not sure how to handle those things, especially because I don’t use typescript.

I think we can revert `unist-ts` -> `unist`, and remove the repo tho.

Reply:
@wooorm I think I'm missing something.  Who's consuming the type definitions and what are they using them for?

Reply:
Okay, I got some context offline and this pattern is more common than I realized.  Apparently, having the type definitions here (and not in their own GH repo) is the way to go.

Reply:
@hrajchert Sweet, could you open an issue for that over on unist-ts, or maybe a PR adding some examples of how it could be used?

Reply:
Sure, I have some busy days ahead but I'll do it as soon as I can, thanks for your help